### ISSUE TITLE
tests_views.py:648-649: Resurrect...

### ISSUE THREAD

**Issue by 0pdd** [fidals/stroyprombeton#492]
The puzzle `455-7b29044c` from #455 has to be resolved: 

https://github.com/fidals/stroyprombeton/blob/bc0f7a6b62c28515ad5e06bd11e6e21812b30a2d/stroyprombeton/tests/tests_views.py#L648-L649

The puzzle was created by duker33 on 11-Mar-19. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0pdd** [comment]
@artemiyds2 @duker33 please pay attention to this new issue.

**Comment by duker33** [comment]
`1`, because this inconsistent data can affect another parts of the system

**Comment by duker33** [comment]
django creates test db named `test_stb_dev`.
Before fixtures loading at setUpClass this django ORM shows this strange data:
```
In [1]: TagGroup.objects.all().count()
Out[1]: 8

In [2]: TagGroup.objects.filter(tags__isnull=True).count()
Out[2]: 8
```

But psql says the same:
```
test_stb_dev=# SELECT count(*) FROM stroyprombeton_taggroup;
 count
-------
     8
(1 row)

test_stb_dev=# SELECT name FROM stroyprombeton_taggroup;
        name
--------------------
 Серия
 Длина
 Ширина
 Высота
 Масса
 Объём
 Внешний диаметр
 Внутренний диаметр
(8 rows)
```

It's strange `*_taggroup` table state for the newly created database

**Comment by duker33** [comment]
find out: seems it's our migrations

**Comment by duker33** [comment]
blocked by https://github.com/fidals/stroyprombeton/issues/531
